website/docs/execution-node/fee-recipient.md

```diff
@@ -1,83 +1,192 @@
+    <strong>Note</strong>: JSON should be delivered as a JSON payload, not as a JSON file. Your client will issue a GET request and expects the response <code>Content-Type</code> header to be <code>application/json</code>.
+    </td>
+  </tr>
+</table>
 
-The following configuration options are available:
+<br />
 
-| Flag                        | Example                                                                                                                                                                                                                                              |
-|-----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `suggested-fee-recipient`   | Sets a default eth address for ALL validators.  Example: `--suggested-fee-recipient=0x046Fb65722E6b0000012BFEBf6177F1D2e9758D9`   Note: This setting overrides the two `config` options below. If you set this, the `config` options will be ignored |
-| `fee-recipient-config-file` | Sets the file location for the Fee Recipient JSON configuration.   Example: `--fee-recipient-config-file=./fee_recipient_config.json`                                                                                                                |
-| `fee-recipient-config-url`  | Sets a URL for a remote Fee Recipient JSON configuration.  Example: `--fee-recipient-config-url=http://example.com/api/getFeeRecipient`                                                                                                              |
 
-**Note: suggested fee recipient flag will override the configuration file. Do not use these flags together, only use one of these flags to provide fee recipient information.**
+An example invocation: `./prysm.sh validator --suggested-fee-recipient=0x046Fb65722E6b0000012BFEBf6177F1D2e9758D9`. 
 
-### Fee Recipient Config File 
+If you don't see any errors after issuing one of the above commands, your fee recipient address has been successfully configured.
 
-The following JSON file format will be used for defining validator public key to eth address.
+<br />
 
-`fee-recipient-config-file` flag requires this json file provided through file path.
-`fee-recipient-config-url` flag requires this json format provided through url.
+#### Fee Recipient JSON Config File
 
-This will allow you to map your validators to corresponding eth addresses or generally cover the remaining keys with a default.
+If you use either `fee-recipient-config-file` or `fee-recipent-config-url` to specify your fee recipient address, your JSON configuration should follow this schema:
 
 ```
 {
   "proposer_config": {
     "0xa057816155ad77931185101128655c0191bd0214c201ca48ed887f6c4c6adf334070efcd75140eada5ac83a92506dd7a": {
```

Comment:
I have no idea if these fee recipients and addresses are real peop's accounts or not, wondering if we can placeholder them in a good way.

Reply:
Great thought. I just pushed an update so all addresses start with `0x01234567` to make it clear that they're placeholders.

This pattern has been documented here: https://www.notion.so/prysmaticlabs/Patterns-13002df64d454c16b076151694491fc8